lib/sensu-plugin/utils.rb

```diff
@@ -33,6 +33,68 @@ def read_event(file)
         exit 0
       end
 
+      ##
+      #  Helper method to convert Sensu 2.0 event into Sensu 1.4 event
+      #    This is here to help keep Sensu Plugin community handlers working
+      #    until they natively support 2.0
+      ##
+      def event_2to1
+        if @event.key?('entity') && @event['client'].empty?
```

Comment:
Should this be `@event['client'].nil?`? Does empty cover `nil`?

Reply:
for the normal use case it definitely needs to be empty... as read_event will set client to {} if client doesn't exist.  I'll update to check for nil? || empty? to cover all cases.

Reply:
`empty?` does not cover `nil.nil?`